Sources/StreamChat/Models/Channel.swift

```diff
@@ -364,3 +364,150 @@ public struct ChannelCapability: RawRepresentable, ExpressibleByStringLiteral, H
     /// Ability to create a call.
     public static var createCall: Self = "create-call"
 }
+
+public extension ChatChannel {
+    /// Can the current user ban members from this channel.
+    var canBanChannelMembers: Bool {
+        ownCapabilities.contains(.banChannelMembers) == true
```

Comment:
The ` == true` part is a leftover from the conversion from controller to channel, right?